circuits/verify.circom

```diff
@@ -158,16 +158,16 @@ template PointEqual() {
   
   for(i=0; i<10; i++) {
     // (P[0] * Q[2]) % p
-    mod[0].a[i] <== mul[0].out[i];
```

Comment:
umm sorry missed this most likely because there are no tests for this module that's why I missed it.

Reply:
Yep, fixing that by including test for verification in actions